modules/sequence/src/main/scala/lucuma/odb/sequence/gmos/longslit/Acquisition.scala

```diff
@@ -68,7 +79,7 @@ sealed trait Acquisition[D, G, F, U] extends SequenceState[D] {
         _  <- optics.roi           := GmosRoi.CentralStamp
         s1 <- scienceStep(10.arcsec, 0.arcsec, ObserveClass.Acquisition)
 
-        _  <- optics.exposure      := (exposureTime * NonNegInt.unsafeFrom(3)).timeSpan
+        _  <- optics.exposure      := lastExpTime(exposureTime)
```

Comment:
Is it okay if the first step exposure time is over 360?

Reply:
The requirement was stated as
`Could we limit the exposure time of step 3 to <= 360 sec?`